app/src/main/java/io/apicurio/registry/rest/RestConfig.java

```diff
@@ -17,8 +17,16 @@ public class RestConfig {
     @Info(category = "rest", description = "Skip SSL validation when downloading artifacts from URL", availableSince = "2.2.6-SNAPSHOT")
     boolean downloadSkipSSLValidation;
 
+    @ConfigProperty(name = "registry.rest.artifact.deletion.enabled", defaultValue = "true")
+    @Info(category = "rest", description = "Enables artifact version deletion", availableSince = "2.4.2-SNAPSHOT")
+    boolean artifactVersionDeletionEnabled;
```

Comment:
We should use dynamic configuration property here, with a disclaimer in the description, so it works in a multi-tenant environment. Similarly to enabling basic auth.

Reply:
Fixed